<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>519</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>519</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>117</y>
     <width>582</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 0);
font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Загруженная таблица</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>328</y>
     <width>582</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(170, 170, 0);
font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Итоговая таблица</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>223</y>
     <width>273</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Команда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>117</y>
     <width>273</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;
</string>
   </property>
   <property name="text">
    <string>Исходные ячейки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>333</y>
     <width>273</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Итоговые ячейки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>430</y>
     <width>214</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 20px;
text-align: center;

color: #FFFFFF
;background: #575757;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_file_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>214</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #575757;
border-radius: 15px;

font-family: 'Gilroy';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 20px;
text-align: center;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>выбрать другой файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>161</y>
     <width>204</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(241, 241, 241);
border-radius: 13px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>271</y>
     <width>204</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(241, 241, 241);
border-radius: 13px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>376</y>
     <width>204</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(241, 241, 241);
border-radius: 13px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>165</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F1F1F1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>165</y>
     <width>316</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F1F1F1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>165</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F1F1F1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>206</y>
     <width>251</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>206</y>
     <width>316</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>206</y>
     <width>92</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_download_btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>460</y>
     <width>214</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #575757;
border-radius: 15px;

font-family: 'Gilroy';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 20px;
text-align: center;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>скачать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
